ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -267,8 +267,8 @@ tests:
       TAG_VERSION: '@4.14'
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-aws-ipi
-- as: aws-ipi-compact-p3-f28-destructive
+    workflow: cucushift-installer-rehearse-aws-ipi-custom-security-groups
```

Comment:
We are also testing worker customer sg, sounds like a compact cluster is not very proper for the testing?

Reply:
Ah, good point, let me choose another one to combine